### DISCUSSION TITLE
Any interest in an Interlingua course?

### DISCUSSION THREAD

**Discussion by kraileth** [kantord/LibreLingo#2821] [Q&A]
Just stumbled across this project and it looks great, especially the community-owned part! I've had the idea of creating a language course for some time. Couldn't help but had a good laugh when I tried out **Spanish-from-English** to get a first impression of LibreLingo. Why? Because the first two topics that my material started with are - animals and food! :laughing:

A short introduction to the language since it's not really well known today (first in _ia_ then the same in _en_):

Interlingua o "le Latino moderne" es un utensile per le communication international. Illo es un lingua auxiliar qui es multo facile. Le grammatica es artificial, minimal e totalmente regular con zero exceptiones. Ma le vocabulario componer se de parolas natural de linguas romanic.

Interlingua or "the modern Latin" is a tool for international communication. It is an auxiliary language which is very easy. The grammar is artificial, minimal and completely regular with zero exceptions. The vocabulary however consists of natural words from romance languages.

For anybody who's a native speaker of a romance language Interlingua is meant to qualify tongue-in-cheek as "the language you didn't know you know" and should be understandable without learning it first. It can be understood to a very high degree by English native speakers and to a lesser (but still around 50%) by native speakers of Germanic or Slavic language families due to the high amount of Latin-derived words that have survived in the European languages to this day.

I'm interested in lifelong learning and think that it is a shame how we as a society frequently treat the elderly (as in "you did a life's worth of work, now get lost as you've turned into a burden!"). I thought that this language could be ideal for a free course which is meant to keep the brain going with learning since it's simple and people are likely to recognize many words from what they encountered from their native languages, from foreign languages they might have learned, from travels and such. A LibreLingo course would of course have a different target group than what I had in mind for "Interlingua per le antenates" ("Interlingua for the elderly"), a wider scope and above all feature a much more modern representation (my material was for a "on paper" course).

Now my question is: Would there be any interest in having **Interlingua-from-English** or **Interlingua-from-German** on the platform? If yes, I'd ask around in the Interlingua community whether I could get help especially with reviewing material as I'm only learning the language myself.

**Comment by kantord** [comment]
> Now my question is: Would there be any interest in having Interlingua-from-English or Interlingua-from-German on the platform? If yes, I'd ask around in the Interlingua community whether I could get help especially with reviewing material as I'm only learning the language myself.

yes, there is interest in including any course, as long as you are willing to create this course and maintain it, or find maintainers for it
